.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
I think you have committed your IDEA files by accident, you may want to remove them.

Reply:
OK